examples/docker/config/config-ssl.sql

```diff
@@ -0,0 +1,10 @@
+ALTER SYSTEM SET ssl_cert_file TO '/etc/ssl/certs/server.crt';
```

Comment:
I don't think baking certs into the image is the right approach, even for docker compose. 

What we need is a no-SSL compose connection to postgres like we have it currently, for ease of local developments. 

We  _also_  need a way to configure client certs to connect to a _cloud_ instance of postgres.